### ISSUE TITLE
Using custom icons 

### ISSUE THREAD

**Issue by drdpedroso** [pregiotek/ion-floating-menu#17]
Is possible to use a custom icon? something like this:

```
<ion-floating-item icon="&#xE53E;"></ion-floating-item>
```

or

```
<ion-floating-item>&#xE53E;</ion-floating-item>
```

**Comment by stephenlane** [comment]
Similar question, is it possible to use images?

**Comment by drdpedroso** [comment]
@stephenlane the way the directive is implemented, is not possible. I made a feel changes for using images and material-design icons (only for simple buttons, not menus)
https://github.com/drdpedroso/ionic-float-button

**Comment by stephenlane** [comment]
@drdpedroso okay no problem. Thanks

**Comment by stephenlane** [comment]
@drdpedroso I have changed the library to work with images, it is crude and works for me but I can share if you wish.

**Comment by drdpedroso** [comment]
@stephenlane Sure! would be nice

**Comment by stephenlane** [comment]
JS file change the last template to include an image tag

```
        .directive('ionFloatingItem', function () {

            return {
                restrict: 'E',
                require: ['^ionFloatingMenu'],
                scope: {
                    click: '&?',
                    icon: '@',
                    buttonColor: '@?',
                    buttonClass: '@?',
                    iconColor: '@?',
                    text: '@?',
                    image: '@?',
                    textClass: '@?'},
                template:
                        '<li ng-click="click()" ng-class="buttonClass" ng-style="{\'background-color\': buttonColor }">' +
                        '<img class="homeIcons" src="{{image}}"/><span ng-if="text" class="label-container"><span class="label" ng-class="textClass" ng-bind="text"></span></span><i class="icon menu-icon" ng-class="{ \'{{icon}}\' : true}" ng-style="{\'color\': iconColor }"></i>' +
                        '</li>',
                replace: true,
                controller: function ($scope) {
                    $scope.buttonColor = $scope.buttonColor || '#FF7900';
                    $scope.iconColor = $scope.iconColor || '#fff';
                }
            };
        });

```

From the front end it can be used as follows

`<ion-floating-item image="img/sampe.png"></ion-floating-item>`

**Comment by ennedigi** [comment]
Fixed in v1.03
